assets/scss/custom.css

```diff
@@ -0,0 +1,3 @@
+.home-section#sponsors {
```

Comment:
The filename should be `custom.scss`, I think once that is in place it should work

Reply:
OK. I made that change locally and it works.